hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -80,8 +80,8 @@ def combine(ts):
                             lambda old_to_new: tmp.data[i].__entries.map(lambda e: renumber_entry(e, old_to_new)),
                             hl.range(0, hl.len(tmp.data[i].alleles)).map(
                                 lambda j: combined_allele_index[tmp.data[i].alleles[j]])))),
-            hl.dict(hl.range(0, hl.len(tmp.alleles)).map(
-                lambda j: hl.tuple([tmp.alleles[j], j])))))
+            hl.dict(hl.range(1, hl.len(tmp.alleles) + 1).map(
```

Comment:
This large inline fn is a bit confusing to me. Could you help me understand what it is doing?